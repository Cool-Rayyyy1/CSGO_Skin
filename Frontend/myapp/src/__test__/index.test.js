import React from 'react';
import renderer from 'react-test-renderer';
import MainPage  from "../pages/index"


it ("snapshot testing for profile screen", () => {
    const tree = renderer.create(
        <MainPage></MainPage>
    ).toJSON();
    expect(tree).toMatchSnapshot();
})
