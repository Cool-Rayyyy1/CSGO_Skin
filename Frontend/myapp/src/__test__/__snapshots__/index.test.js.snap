// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing for profile screen 1`] = `
<div
  className="App"
>
  <header
    className="lan"
  >
    <div
      className="button3"
    >
      <div
        className="ui left action input"
      >
        <button
          className="ui black icon left labeled button"
          onClick={[Function]}
        >
          <i
            aria-hidden="true"
            className="cart icon"
            onClick={[Function]}
          />
          Checkout
        </button>
        <input
          defaultValue="0.0"
          onChange={[Function]}
          placeholder="Search..."
          type="text"
        />
      </div>
      <div
        className="ui icon input"
      >
        <input
          onChange={[Function]}
          placeholder="Search User..."
          type="text"
        />
        <i
          aria-hidden="true"
          className="search circular inverted link icon"
          onClick={[Function]}
        />
      </div>
    </div>
    <div
      className="button2"
    >
      <button
        className="ui secondary button"
        onClick={[Function]}
      >
        Main
      </button>
      <button
        className="ui secondary button"
        onClick={[Function]}
      >
        Search
      </button>
      <button
        className="ui secondary button"
        onClick={[Function]}
      >
        Inventory
      </button>
    </div>
  </header>
  <div
    className="MainPageImage"
  />
  <div />
</div>
`;
